app/controllers/admin/groups_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Admin::GroupsController < ApplicationController
```

Comment:
It doesn't really matter, it will be in a separate commit anyway so it's up to you ! :-)